include/SH3/arc/vfile.hpp

```diff
@@ -14,6 +14,24 @@
 #include "SH3/arc/types.hpp"
 #include <ios>
 
+/**
+ *  Opens a handle to a 'virtual file' within an arc section.
+ *
+ *  After the handle to the file is open, it is possible to seek and rewind
+ *  to specific parts of the file using @ref Seek().
+ *
+ *  Each read sets the @ref read_error to indicate whether a partial read was performed
+ *  or that the end of file was encountered (meaning that @ref fpos + len was equal to @ref fsize.
```

Comment:
``@ref fpos `+ len` `` puts the "+ len" parts in typewriter font, which is normally used for code snippets.
Sadly there doesn't seem to be a way to use `@ref` within that, so keep it outside the quotation.